packages/gamut/src/ProgressBar/index.tsx

```diff
@@ -170,14 +174,13 @@ export const ProgressBar: React.FC<ProgressBarProps> = ({
 }) => {
   return (
     <ProgressBarWrapper
-      aria-label={`Progress: ${percent}%`}
       aria-live="polite"
-      role="figure"
       border={bordered ? 'bordered' : 'basic'}
       size={size}
       variant={variant}
       backgroundOverride={pattern ? 'pattern' : 'none'}
     >
+      <ScreenReaderText>{`Progress: ${percent}%`}</ScreenReaderText>
```

Comment:
This is a prop on Text

```suggestion
      <Text screenreeder>{`Progress: ${percent}%`}</Text>
```

Reply:
😲

Reply:
@codecaaron will as="label" work on Text?

Reply:
Go ahead and add its as an empty as variant on Text `as` variants.

Same goes for `figcaption`.